app/lib/utils.dart

```diff
@@ -36,10 +37,14 @@ Future<List<String>> listTarball(String path) async {
 }
 
 Future<String> readTarballFile(String path, String name) async {
-  final result = await Process.run('tar', ['-O', '-xzf', path, name]);
+  final result = await Process.run(
+    'tar',
+    ['-O', '-xzf', path, name],
+    stdoutEncoding: null,
```

Comment:
In `pubspec` the most probable cause for weird characters will be the author's name, the description or a comment block. I see very little reason to throw on these.